js/app.js

```diff
@@ -43,3 +48,42 @@ document.getElementById("starting-items").textContent = getRandomWithSeed(
   .map(items => `C${items.id} ${items.name}`)
   .join(", ");
 document.getElementById("end-room").textContent = endingRoom;
+document.getElementById("generate-seed").addEventListener("click", function() {
+  newUrl = `?seed=${generateANewSeed()}`;
+  redirect(newUrl);
```

Comment:
I think this would be neater just as this;

```JavaScript
window.location = `?seed=${generateANewSeed()}`
```

I don't think a `redirect` function is needed, as it is only a single line and only used in one place too (and setting `window.location` is a well known way to perform a redirect).

In regards to using `window.location.replace` instead of `window.location` - `.replace` actually replaces the current history item, meaning the user cannot press the back button to go back to the previous page (e.g. if they pressed generate 3 times, they could not use the back button to view previously generated seeds).

Reply:
Good idea, didn't know about that cheers!